app/controllers/api/v1/notes_controller.rb

```diff
@@ -0,0 +1,74 @@
+module Api
+  module V1
+    class NotesController < ApplicationController
+      def index
+        return render_long_page_size if page_size.to_i > max_page_size
+        return render_invalid_type if type.present? && invalid_note_type
+        render json: notes, status: :ok, each_serializer: BriefNoteSerializer
+      end
+
+      def show
+        render json: note, status: :ok, serializer: NoteSerializer
+      end
+
+      private
+
+      def notes
+        Note.filtered filter_params, order, page, page_size
+      end
```

Comment:
Bieeen, si bien ruby acepta que no usemos paréntesis, por convención todos los usamos 🤗 mantengamos ese criterio
```suggestion
      def notes
        Note.filtered(filter_params, order, page, page_size)
      end
```

Reply:
Anotado!